### DISCUSSION TITLE
FlashBoy?

### DISCUSSION THREAD

**Discussion by ArchieDxncan** [sanni/cartreader#476] [General Chat]
Hello, I was wondering if my FlashBoy would work with this project, I assume the answer is no but just need to make sure!

(I clicked on this at the start of a LTT video assuming it allowed you to play the roms using the official cartridge, please close if I was wrong!)

**Comment by kamalawala** [comment]
Could you elaborate please? A quick search brings up a Game Boy cartridge flasher/dumper and a Virtual Boy flash cartridge. The Game Boy flasher description says it has flash cartridges but I could only find a reddit post showing one, and a comment suggests it may be a jrodrigo cartridge clone. Some pictures of the front and the back of the PCB would be very helpful.
